contracts/registry/PocketRegistryDelegate.sol

```diff
@@ -19,6 +15,7 @@ contract PocketRegistryDelegate {
     owner = msg.sender;
```

Comment:
Does this means that anyone can create a Registry? Should we restrict that?

Reply:
Anyone can create a Registry, this piece of code just tells that WE own the right to delegate.